apps/wallet/src/ui/app/shared/text/index.tsx

```diff
@@ -43,9 +43,13 @@ const textStyles = cva([], {
 			'steel-darker': 'text-steel-darker',
 			hero: 'text-hero',
 			'hero-dark': 'text-hero-dark',
+			'hero-darkest': 'text-hero-darkest',
 			'success-dark': 'text-success-dark',
 			'issue-dark': 'text-issue-dark',
 		},
+		opacity: {
```

Comment:
Instead of doing this can you apply this at the parent and depend on the cascade? I don't want to infinitely inflate these props

Reply:
That does not work since `text-*` automatically applies `--tw-text-opacity: 1;` at the innerchild, so the styling from parent does not cascade down.


<img width="399" alt="Screenshot 2023-09-29 at 9 19 10 AM" src="https://github.com/MystenLabs/sui/assets/127577476/8d634944-9fea-4a4f-9fa6-3edc685a7e46">